release notes/v0.55.0.md

```diff
@@ -0,0 +1,205 @@
+k6 `v0.55.0` is here 🎉! This release includes:
+
+- ⚠️ The deprecated StatsD output has been removed.
+- ⚠️ The experimental `k6/experimental/tracing` module has been removed.
+- URL grouping support in the browser module.
+- Top-level `await` support.
+- Complete RSA support for `k6/experimental/webcrypto`.
+
+## Breaking changes
+
+### `k6/experimental/tracing` module removed [`#3855`](https://github.com/grafana/k6/pull/3855)
+
+The experimental `k6/experimental/tracing` module has been removed, in favor of a replacement [jslib](https://jslib.k6.io/http-instrumentation-tempo/), please look at the [docs on how to transition](https://grafana.com/docs/k6/latest/javascript-api/jslib/http-instrumentation-tempo/#migration-from-k6experimentaltracing), [`#3855`](https://github.com/grafana/k6/pull/3855).
+
+### StatsD output removed [`#3849`](https://github.com/grafana/k6/pull/3849)
+
+The StatsD output was deprecated in k6 v0.47.0 and is now removed. You could still output results to StatsD using the community xk6 extension [LeonAdato/xk6-output-statsd](https://github.com/LeonAdato/xk6-output-statsd). Thanks, @LeonAdato for taking over the extension!
+
+### `open` will have a breaking change in the future.
+
+Currently `open` opens relative files based on a strange root the same `require` before it was fixed around ESM. As part of actually making k6 consistent `open` among other functions like it will start to be relative in the same way as imports and `require` are.
```

Comment:
```suggestion
Currently, `open` opens relative files based on an unusual root, similar to how `require` behaved before it was updated for ESM compatibility. To make k6 more consistent, `open` and other functions like it will start handling relative paths in the same way as imports and `require`.
```

WDYT?